projects/distributed-tracing/src/shared/graphql/request/handlers/traces/traces-graphql-query-handler.service.ts

```diff
@@ -81,6 +86,12 @@ export class TracesGraphQlQueryHandlerService implements GraphQlQueryHandler<Gra
       })
     );
   }
+
+  private resultUnits(specification: Specification): Observable<string | undefined> {
+    return this.metadataService
+      .getAttribute('API_TRACE', specification.name)
```

Comment:
the scope should come off the requested trace type

Reply:
Fixed.